src/App.tsx

```diff
@@ -13,40 +14,42 @@ import { TimelineId } from "./TimelineID/TimelineId";
 export const App: FunctionComponent = () => {
     return (
         <Router>
-                <nav>
-                    <ul className={styles.navbarUl}>
-                        <li className={styles.navbarLi}>
-                            <Link to="/">Home</Link>
-                        </li>
-                        <li className={styles.navbarLi}>
-                            <Link to="/rover_explorer">Rover explorer</Link>
-                        </li>
-                        <li className={styles.navbarLi}>
-                            <Link to="/timeline/opportunity">Opportunity</Link>
-                        </li>
-                        <li className={styles.navbarLi}>
-                            <Link to="/timeline/spirit">Spirit</Link>
-                        </li>
-                        <li className={styles.navbarLi}>
-                            <Link to="/timeline/curiosity">Curiosity</Link>
-                        </li>
-                        <li className={styles.navbarLi}>
-                            <Link to="/timeline/perserverance">Perserverance</Link>
-                        </li>
-                    </ul>
-                </nav>
-
-                <Switch>
-                    <Route path="/rover_explorer">
-                        <RoverExplorer />
-                    </Route>
-                    <Route path="/timeline/:id">
-                        {<TimelineId />}
-                    </Route>
-                    <Route path="/">
-                        <Home />
-                    </Route>
-                </Switch>
+            <nav>
+                <ul className={styles.navbarUl}>
+                    <li className={styles.navbarLi}>
+                        <Link to="/">Home</Link>
+                    </li>
+                    <li className={styles.navbarLi}>
+                        <Link to="/rover_explorer">Rover explorer</Link>
+                    </li>
+                    <li className={styles.navbarLi}>
+                        <Link to="/timeline/opportunity">Opportunity</Link>
+                    </li>
+                    <li className={styles.navbarLi}>
+                        <Link to="/timeline/spirit">Spirit</Link>
+                    </li>
+                    <li className={styles.navbarLi}>
+                        <Link to="/timeline/curiosity">Curiosity</Link>
+                    </li>
+                    <li className={styles.navbarLi}>
+                        <Link to="/timeline/perserverance">Perserverance</Link>
+                    </li>
+                </ul>
+            </nav>
+                 <Switch>
+                <Route path="/rover_explorer">
+                    <RoverExplorer />
+                </Route>
+                <Route path="/timeline/:id">
+                    {<TimelineId />}
+                </Route>
+                <Route path="/">
+                <Link to="/rover_explorer">
+                <button className= {styles.marsbutton} >
```

Comment:
we don't really want a button inside a link.
- links are for changing the page someone is on
- buttons are for submitting forms or triggering behaviour on a page

in this case I suspect we want a link but not a button.